src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -127,4 +132,34 @@ public void appendsAttributesFromInputStream() throws Exception {
         );
     }
 
+    /**
+     * Manifests can read values from several manifests.
+     * @throws Exception If something goes wrong
+     * @since 2.0
+     */
+    @Test
+    public void readsValuesFromSeveralManifests() throws Exception {
+        final List<InputStream> streams = Arrays.asList(
+                this.getClass().getResourceAsStream("test.mf"),
+                this.getClass().getResourceAsStream("another-test.mf")
+        );
+        final Mfs streamManifests = new StreamsMfs(streams);
+        final Collection<String> values = Manifests.all(
+                streamManifests,
+                "From-File"
+        );
+        MatcherAssert.assertThat(
+                values.size(),
+                Matchers.equalTo(2)
+        );
+        MatcherAssert.assertThat(
+                values.contains("some test attribute"),
+                Matchers.equalTo(true)
+        );
+        MatcherAssert.assertThat(
+                values.contains("another test attribute"),
+                Matchers.equalTo(true)
+        );
+    }
+
```

Comment:
@nesterov-n we need more tests here: 0 manifests, 2 manifests but only one with the key etc.